### DISCUSSION TITLE
Readme showing to non-existing composer package?

### DISCUSSION THREAD

**Discussion by Norgul** [roadrunner-server/roadrunner#1005] [Q&A] [answered]
Readme says that it is possible to install package via CLI by running `composer require roadrunner-server/roadrunner:v2.0 nyholm/psr7 `, however it throws 

> [InvalidArgumentException]
> Could not find a matching version of package roadrunner-server/roadrunner. Check the package spelling, your version constraint and that the package is available in a stability which matc
> hes your minimum-stability (dev).

Looking at Packagist, there is only `spiral` repo registered. Am I missing something?

**Comment by rustatian** [comment, marked as answer]
> 

Hey @Norgul. Yes, the `roadrunner-server` org contains only Go packages. All PHP composer dependencies remain as they were before moving RR to the new org. So -> `composer require spiral/roadrunner:v2.0 ...` is the correct command.

**Reply by Norgul** [reply]
Thanks

**Reply by rustatian** [reply]
I've updated the Readme, thank you 👍🏻
